db_instance/versions.tf

```diff
@@ -0,0 +1,11 @@
+terraform {
+  required_version = "1.0.4"
+  required_providers {
+    mcs = {
+      source = "MailRuCloudSolutions/mcs"
```

Comment:
Can you explain how this custom provider works?

Reply:
this is custom provider written by Vk cloud team, it convert HCL (terraform) language to the API calls understandable to custom's Vk cloud solutions

Reply:
by default providers downloads from the terraform registry:
https://registry.terraform.io/providers/MailRuCloudSolutions/mcs/latest
I will add that link to the Readme